src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -316,6 +318,10 @@ export default class DataTable extends Vue {
       selected: true,
     };
 
+    if (this.config.selectable === 'radio') {
+      this.selectedRows = [];
```

Comment:
```suggestion
      this.selectedRows.length = 0;
```
`ARRAY.length = 0` cleans the existing array, `ARRAY = []` replaces the existing array with another, empty one.